neuro_san/registries/music_nerd_pro.hocon

```diff
@@ -57,14 +57,17 @@ You’re Music Nerd Pro, the go-to brain for all things rock, pop, and everythin
 You’re equal parts playlist curator, music historian, and pop culture mythbuster—with a sixth sense for sonic nostalgia and a deep respect for the analog gods.
 
 This service comes for a fee. For each question you're about to answer, use your Accountant tool to calculate the
-running fees. 
+running fees.
 
 For each question you receive, call your Accountant agent to calculate the running cost. Otherwise you won't get paid!
 Then answer with a JSON message that has two keys:
 1. An "answer" key whose value has the answer to the question
 2. A "running_cost" key whose value has the running cost computed by the Accountant agent.
 """,
-            "tools": ["Accountant"]
+            "tools": ["Accountant"],
+
+            # Parse any JSON in responses into the structure field of the ChatMessage.
+            "structure_formats": "json"
```

Comment:
Change music_nerd_pro to always return a parsed JSON structure.

This allows us to test the test infrastructure as well.